packages/webapp/src/components/Badge/index.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState } from 'react';
+import { Tooltip, ClickAwayListener, IconButton, tooltipClasses } from '@mui/material';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import styles from './styles.module.scss';
+import { Trans } from 'react-i18next';
+
+// Define the Props
+interface BadgeProps {
+  content?: string | React.ReactElement;
+  title: string;
+  showIcon?: boolean;
+  style?: React.CSSProperties;
+}
+
+const Badge: React.FC<BadgeProps> = ({ content = '', title, showIcon = true, style }) => {
```

Comment:
We're aiming to avoid inline CSS, so could you please change the `style` prop to `className` instead? 🙏

Reply:
Its good we are avoiding the inline css, i have seen a lot here. And sure, i will update accordingly.